learn/safe-core-account-abstraction-sdk/auth-kit.md

```diff
@@ -92,6 +94,27 @@ const safeAuthKit = await SafeAuthKit.init(SafeAuthProviderType.Web3Auth, {
 
 When `txServiceUrl` is provided, the list of associated Safe addresses will be returned as part of the `signIn()` method response.
 
+## Sending/signing transactions or messages using the auth-kit
+
+Once you are logged in using the auth-kit you can get the associated web3 provider using the `safeAuthKit.getProvider()` method. If you want to make any kind of operation afterwards you can use this method in conjunction with your favorite web3 library:
+
+```typescript
+// Using web3
+const web3 = new Web3(safeAuthKit.getProvider());
+
+await web3.eth.sendTransaction(tx);
+await web3.eth.signTransaction(tx);
+await web3.eth.personal.sign(message, address);
```

Comment:
Might be useful to define `message` so people know what type of message is expected.

Is it a regular string? hexadecimal string? dictionary etc.

```suggestion
const message = "hello world"
const address = "0x..."
await web3.eth.personal.sign(message, address);
```